include/oneapi/dpl/pstl/hetero/dpcpp/sycl_defs.h

```diff
@@ -234,6 +234,13 @@ constexpr __target __target_device =
     __target::global_buffer;
 #endif
 
+constexpr __target __host_target =
+#if _ONEDPL_LIBSYCL_VERSION >= 60200
```

Comment:
I got this version by finding the commit which marked `host_buffer` as deprecated in libsycl, which is https://github.com/intel/llvm/commit/a8401952 and noting that the version of the library at that commit was 6.2.0.